jwst/stpipe/utilities.py

```diff
@@ -51,6 +52,7 @@
     'SystemCall',
 ]
 
+NOT_SET = "NOT SET"
```

Comment:
Should we add COMPLETE and SKIPPED here too?  It seems odd to keep them as strings if NOT_SET is a constant.

Reply:
yeah, we can definitely do this, and it was discussed between Brett and me.  The question was basically whether it's worth doing that now in JWST, or if we should do that later along with other changes to stpipe.  Not sure we came to a definitive conclusion about that.  If that is done here, then it will touch (albeit extremely minorly) basically every step in the pipeline

Reply:
I think we wouldn't necessarily have to change all the steps yet, since we'd set it to the same string that is currently being checked anyway, but I take your point.

Reply:
I did add these as variables in the changes that were required to support the logic to return COMPLETE for at least one completion and SKIPPED only for all skipped, but I didn't change these everywhere.  My suggestion would be that if/when we move this stuff into stpipe, that would be a good time to make that change and import the constants from stpipe.